docs/releases/readme.md

```diff
@@ -45,6 +45,10 @@ _Outcome_: **Team is aware of release and in agreement about what fixes & featur
     -   For _major_ and _minor_ releases, create branch: `release/X.X`.
     -   For _patch_ releases, the branch should already exist.
 
+#### Create a release pull request
```

Comment:
Can we use a github action to detect a major/minor update and create this for us?

Reply:
Yeah I have plans for that, but I want to get this merged first so we have the process in place _until_ the action is done.

What I'd like to do is have a pull automatically created when we push a specifically named branch (i.e. `release/x.x`).